src/main/java/uk/ac/ebi/atlas/download/FileDownloadController.java

```diff
@@ -116,6 +117,9 @@ public FileDownloadController(ExperimentFileLocationService experimentFileLocati
             }
         }
 
+        var fileTypeCheckList = ImmutableSet.<ExperimentFileType>builder();
+        fileTypeIds.stream().forEach(fileTypeId -> fileTypeCheckList.add(ExperimentFileType.fromId(fileTypeId)));
+
```

Comment:
I guess formatting code according to following will increase the readability of code.
`fileTypeIds`
           `.stream()`
           `.foreach(fileTypeId ->`
                   `fileTypeChecklist.add(...)`

Reply:
Refactored.